src/main/java/homework_2/random_chars_table/RandomCharsTable.java

```diff
@@ -0,0 +1,106 @@
+package homework_2.random_chars_table;
+
+import java.util.Random;
+
+import static homework_2.IOMod.*;
+
+public class RandomCharsTable {
```

Comment:
![image](https://user-images.githubusercontent.com/58872509/126909996-3a5bbcb3-0d85-445b-9158-160a12e12fd7.png)
Please fix according to Homework